XYZDrivers/src/main/java/com/xyzdrivers/controllers/MembershipStatusController.java

```diff
@@ -0,0 +1,99 @@
+package com.xyzdrivers.controllers;
+
+import com.xyzdrivers.models.Member;
+import com.xyzdrivers.repositories.MembersRepo;
+import com.xyzdrivers.services.ResponseService;
+import java.io.IOException;
+import javax.inject.Inject;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ *
+ * @author arthur
+ */
+public class MembershipStatusController extends HttpServlet {
```

Comment:
Pretty sure I'd already written the service to upgrade members - so it's probably worth either removing mine, or integrating them, or whatever is best. Your code removes the need for a service so probably best to stick with yours.